cdxev/auxiliary/schema/bom-1.5-custom.schema.json

```diff
@@ -246,7 +246,7 @@
               "properties": {
                 "components": {
                   "type": "array",
-                  "items": {"$ref": "#/definitions/component"},
+                  "items": {"$ref": "#/definitions/default_component"},
```

Comment:
I would prefer a name like `tools_component`, because it is used within this array, default sounds odd.